addons/web_editor/static/src/components/media_dialog/video_selector.js

```diff
@@ -256,9 +256,11 @@ export class VideoSelector extends Component {
             div.innerHTML = `
                 <div class="css_editable_mode_display"></div>
                 <div class="media_iframe_video_size" contenteditable="false"></div>
-                <iframe frameborder="0" contenteditable="false" allowfullscreen="allowfullscreen"></iframe>
+                <iframe frameborder="0" contenteditable="false" allowfullscreen="allowfullscreen" title="Video"></iframe>
             `;
-            div.querySelector('iframe').src = video.src;
+            const iframeEl = div.querySelector("iframe");
+            iframeEl.src = video.src
+            iframeEl.loading = "lazy";
```

Comment:
Nitpick but you could directly add it as an attribute in the HTML of the iframe right above, no ? :thinking:

Reply:
indeed :)